nautobot/extras/views.py

```diff
@@ -1672,6 +1675,7 @@ def post(self, request, pk):
                 scheduled_job.approved_by_user = request.user
                 scheduled_job.approved_at = timezone.now()
                 scheduled_job.save()
+                publish_event(topic="nautobot.jobs.approval.approved", payload=publish_event_payload)
```

Comment:
Don't we need to regenerate the payload here since we just updated the scheduled_job in the three lines above?